accounts/views.py

```diff
@@ -77,6 +77,17 @@ def crash_me(request):
     raise Exception
 
 
+def check_username(request):
+    username = request.GET.get('username', None)
+    res = {"result": False}
```

Comment:
Yes, that is much better. I keep the brackets instead of 'dict' because is slightly faster: http://stackoverflow.com/questions/1929274/in-python-use-dict-with-keywords-or-anonymous-dictionaries